docs/usage.rst

```diff
@@ -517,5 +517,25 @@ Methods for manipulating the graph
     :param str module: The name of a module, for example ``'mypackage.foo'``.
     :return: bool
 
+.. _module_expressions:
+
+Module expressions
```

Comment:
🍷 This text pretty much copied from import linter docs here https://import-linter.readthedocs.io/en/stable/contract_types.html#wildcards